### PR TITLE
Merge from vbtest-branch

### PR DESCRIPTION
**Pull request by vb186017**
This is linked to #150
- [x] Pull request approved

### CODE REVIEW

**Comment by annavester** [comment]
@vb186017 code reviewed and is ready to merge. :+1:

File: branchMarkDown.md [hunk 1]
```diff
@@ -0,0 +1 @@
+## This is the mark down file of the branch
```
Comment (metherton) [inline branchMarkDown.md:1]: @vb186017 nice fix :+1:

Comment (satishsaley) [inline branchMarkDown.md:1]: @vb186017 could you please mention the branch name :+1: :trollface:  :8ball:

Comment (cherrybee) [inline branchMarkDown.md:1]: I like this file :smile:

**Comment by vb186017** [comment]
Approved pull request. :shipit:

**Pull request merged**